picard/formats/id3.py

```diff
@@ -351,8 +351,9 @@ def _load(self, filename):
                     name += ':%s' % frame.desc
                 metadata.add(name, frame.text)
             elif frameid == 'SYLT' and frame.type == 1:
-                if frame.format == 0:
-                    raise NotImplementedError("SYLT format 0 is not supported")
+                if frame.format != 2:
+                    log.warning("SYLT formats other than 2 are not supported")
```

Comment:
Do we want to give any more detail to the user e.g. about which file has SYLT tags with invalid format?